NativePlugin/code/build.gradle

```diff
@@ -0,0 +1,50 @@
+apply plugin: 'com.android.library'
+
+android {
+  compileSdkVersion 25
+  buildToolsVersion '25.0.2'
+
+  defaultConfig {
+    externalNativeBuild {
+      cmake {
+        targets 'Earcut'
+        cppFlags '-std=c++11'
+        arguments '-DANDROID_TOOLCHAIN=clang',
+                  '-DANDROID_STL=c++_shared'
+                    cppFlags '-std=c++14',
+                  '-pedantic',
+                  '-fPIC'
+      }
+    }
+  }
+
+  externalNativeBuild {
+    cmake {
+      path '../CMakeLists.txt'
+    }
+  }
+
+  buildTypes {
+    release {
+      externalNativeBuild {
+        cmake.cppFlags'-g0'
+      }
+    }
+  }
+
+  productFlavors {
+    full {
+      externalNativeBuild.cmake.abiFilters 'armeabi-v7a', 'x86'
```

Comment:
I don't think we need a specific product flavor for building all the ABIs in this scenario, we can put this `abiFilters` list directly in `defaultConfig.externalNativeBuild.cmake` above.